src/components/block-button.js

```diff
@@ -0,0 +1,31 @@
+window.blockUser = function(id) {
+  NAF.connection.adapter.publisher.handle.sendMessage({ kind: "block", whom: id });
+  NAF.connection.entities.removeEntitiesOfClient(id);
+};
+
+window.unblockUser = function(id) {
+  NAF.connection.adapter.publisher.handle.sendMessage({ kind: "unblock", whom: id });
```

Comment:
same as above, should be moved to an `unblock(clientId)` function in the adapter.

Reply:
👍